index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en"> 
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Apple</title>
+        <script src="https://kit.fontawesome.com/1f67cb811d.js"></script>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./Assets/stylesheet.css">
+    </head>
+    <body>
+        
+            
+        
+        <main>
+                <nav>
+                    <ul>
+                        <li><a href="#"><i class="fab fa-apple"></i></a></li>
```

Comment:
`REQUIRED`

The home icon (Apple image) seems not to be showing on the page. See below:


|Current|Preffered|
| :-----: | :------: |
| ![current apple home icon](https://user-images.githubusercontent.com/36057474/62106161-6cb5b200-b29c-11e9-9b1f-065c8648a3ea.png)  | ![preffered apple home icon png](https://user-images.githubusercontent.com/36057474/62106180-7a6b3780-b29c-11e9-835d-a3cba6812631.png)  |